code/builders/builder-webpack5/src/loaders/export-order-loader.ts

```diff
@@ -6,16 +6,26 @@ export default async function loader(this: LoaderContext<any>, source: string) {
   const callback = this.async();
 
   try {
+    // Do NOT remove await here. The types are wrong! It has to be awaited,
+    // otherwise it will return a Promise<Promise<...>> when wasm isn't loaded.
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    const [_, exports] = parse(source);
+    const [_, exports = []] = await parse(source);
```

Comment:
FYI you can do `const [, exports = []] = ...` to skip the first entry, and to remove the lint error.

Reply:
TIL 🧠